tools/RAiDER/cli/raider.py

```diff
@@ -442,9 +445,7 @@ def downloadGNSS():
 
 
 ## ------------------------------------------------------------ prepFromGUNW.py
-def calcDelaysGUNW():
-    from RAiDER.aria.prepFromGUNW import main as GUNW_prep
-    from RAiDER.aria.calcGUNW import tropo_gunw_slc as GUNW_calc
+def calcDelaysGUNW(iargs: list[str] = None):
```

Comment:
Following this: https://jugmac00.github.io/blog/testing-argparse-applications-the-better-way/

See earlier functions in this file - I see that Jeremy was already doing this.